### PR TITLE
Add support for Printrbot Neopixel RGBW strip.

### PR DESCRIPTION
**Pull request by cscott**
Connected as described at http://printrbot.com/shop/led-strip/

Based on patch by Kelly Anderson <kelly@xilka.com> at
http://www.xilka.com/printrbot/marlin/1.1.4/20170707/

### CODE REVIEW

**Comment by teemuatlut** [comment]
Would this also work with WS2812 LEDs and not just NeoPixel branded ones?
@Tannoo What happened to your previous addressable LED efforts?

**Comment by Tannoo** [comment]
Mine works for all RGBs, RGB Strips, and any addressable strips that are supported by the fastLED library.

I was going to re-PR it once 2.0 is released.

Got tired of updating it every few days waiting for the possibility of a merge.

**Comment by cscott** [comment]
Presumably it would work with anything Adafruit's NeoPixel library works with.

https://github.com/adafruit/Adafruit_NeoPixel/blob/master/Adafruit_NeoPixel.cpp claims that it works for "a wide variety of WS2811- and WS2812-based RGB LED devices such as Adafruit FLORA RGB Smart Pixels and NeoPixel strips."

**Comment by Tannoo** [comment]
Mine has beed changed (as per @Thinkyhead) to use a single function name for all RGB leds.

**Comment by Tannoo** [comment]
I have been using it on my printer ever since.

File: Marlin/Marlin_main.cpp [hunk 1]
```diff
@@ -981,14 +985,66 @@ void servo_init() {
...
+
+	delay(2000);
+	neo_display(pixels.Color(255, 0, 0, 0)); // red
+	delay(2000);
+	neo_display(pixels.Color(0, 255, 0, 0)); // green
+	delay(2000);
+	neo_display(pixels.Color(0, 0, 255, 0)); // blue
+	delay(2000);
+	neo_display(pixels.Color(0, 0, 0, 255)); // white
+	delay(2000);
```
Comment (thinkyhead) [inline Marlin/Marlin_main.cpp:1013]: This will add 10 seconds onto every reboot and reconnect, won't it?

**Comment by thinkyhead** [comment]
Prepped for merge over at #7340. Please review the changes.

**Pull request closed without merging**